src/Popconfirm/stories/index.stories.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import { Button } from 'antd'
+import { WarningCircle, XCircle, CheckCircle, Info } from 'phosphor-react'
+import { default as Popconfirm, PopconfirmProps } from '../index'
+
+export default {
+  title: 'Popconfirm',
+  component: Popconfirm
+} as ComponentMeta<typeof Popconfirm>
+
+const Template: ComponentStory<typeof Popconfirm> = (args: PopconfirmProps) => {
+  return (
+    <Popconfirm {...args}>
+      <Button type="primary" style={{ margin: '150px' }}>
+        Click Me
+      </Button>
+    </Popconfirm>
+  )
+}
+
+export const Top = Template.bind({})
+
+Top.args = {
+  label: 'Popconfirm',
+  title: 'Are you sure you wish to delete this task',
+  placement: 'top',
+  icon: <WarningCircle size={16} className={'icon-none'} />,
```

Comment:
Warning Icons are part of the design, they should be exported in `Popconfirm/index.tsx` and also in `src/index.tsx`

Reply:
Then you can import them in the storybook